pkgs/by-name/do/dosbox-staging/package.nix

```
@@ -89,29 +90,40 @@ stdenv.mkDerivation (finalAttrs: {
       ]
     );
 
+  outputs = [ "out" "man" ];
+
   postInstall = ''
     install -Dm644 $src/contrib/linux/dosbox-staging.desktop $out/share/applications/
   '';
 
+  # Rename binary, add a wrapper, and copy manual to avoid conflict with
+  # original dosbox. Doing it this way allows us to work with frontends and
+  # launchers that expect the binary to be named dosbox, but get out of the way
+  # of vanilla dosbox if the user desires to install that as well.
```

```suggestion
  # Rename binary, add a wrapper, and copy manual to avoid conflict with
  # original dosbox. Doing it this way allows us to work with frontends and
  # launchers that expect the binary to be named dosbox, but get out of the way
  # of vanilla dosbox if the user desires to install that as well.
```

I've received reverse feedback on this from some other commiter. Maybe @SuperSandro2000 ? (It's hard to remember the source, I remember the issue.)

Like, the comment should be inside the phase because it's a bash comment.

---

I don't like Bash comments.

Roughly speaking, Nix is the high level lang, while Bash is assembly.
Bash comments should be restricted to Bash specificities.
Since this comment is about a decision about installation, it should be on a higher level.

Also, since this is a string that will be passed to the "assembler", Nix will rebuild this if the comment is modified. However, the inferior Bash process will happily ignore the comment.

And we end with a strange situation where the superior process requires an unneeded rebuild to the inferior one!